server/controllers/cohort.js

```diff
@@ -401,31 +380,31 @@ const changeToMember = (req, resp) => {
     (err, doc) => {
       if (err) {
         return resp.status(400).send({
-          message: "Can't set user as a cohort's member."
+          message: "Can't remove owner role."
         });
       }
 
       if (doc) {
         return resp.status(200).send({
-          message: "User has been successfully set as a cohort's member."
+          message: 'User has no owner role.'
         });
       }
 
-      resp.status(404).send({ message: 'Cohort data not found.' });
+      resp.status(400).send({ message: 'Cohort data not found.' });
```

Comment:
please convert those statements to match our overall pattern where we check: 
1. if(err)
2. if( !doc )
3. return status(200)